R/board_connect_bundle.R

```diff
@@ -52,6 +52,7 @@ rsc_bundle_preview_create <- function(board, name, x, metadata, path) {
 }
 
 rsc_bundle_preview_index <- function(board, name, x, metadata) {
+  check_installed("whisker")
```

Comment:
I think it's probably worth leaving whisker in imports, since if you're using Connect, you'll hit this path?